lib/merchant_repository.rb

```diff
@@ -1,2 +1,13 @@
+require_relative 'merchant_parser'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.